website/playground/new-samples/interacting-with-the-editor/listening-to-key-events/sample.js

```diff
@@ -7,4 +7,5 @@ var myBinding = editor.addCommand(monaco.KeyCode.F9, function() {
 	alert('F9 pressed!');
 });
 
-// When cleaning up remember to call myBinding.dispose()
+// You can't dispose `addCommand`
+// If you need to dispose it you might use `addAction` or `registerCommand`
```

Comment:
I don't know all about addCommand / addAction, but why would you need to dispose?

Reply:
I'm using monaco-editor in a NextJS app. 
I need it to be updated every time an specific hook updates. If I don't dispose it the app gets slow pretty quick.
Now I'm just using `addAction` because it can be dispose.
It seams like [it used to return a dispose](https://github.com/microsoft/monaco-editor/issues/942) then it change and someone forgot to update the playground.

Reply:
[Here I opened](https://github.com/suren-atoyan/monaco-react/issues/277) an issue on [monaco-react](https://github.com/suren-atoyan/monaco-react) . At thebegining I thought I could use `addCommand` because in the playground you can see the lines:

```js
// When cleaning up remember to call myBinding.dispose()
```

But, that is not true anymore. That's why I changed.